services/QuillLMS/app/models/translated_text.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: translated_texts
+#
+#  id                 :bigint           not null, primary key
+#  locale             :string           not null
+#  translation        :text
+#  created_at         :datetime         not null
+#  updated_at         :datetime         not null
+#  english_text_id    :integer          not null
+#  translation_job_id :string           not null
+#
+class TranslatedText < ApplicationRecord
+  belongs_to :english_text
+  scope :pending_translation, -> { where(translation: nil) }
+
+  def self.fetch_pending!
+    pending_translation.each(&:fetch_translation!)
+  end
+
+  def fetch_translation!
+    Gengo::SaveTranslatedText.run(translation_job_id)
+  end
```

Comment:
This is a new Ruby thing, so you can take or leave, but there are a few methods in this PR that could be one-liners. 

Note, if you are not used to one line methods from other languages, the new syntax takes a little getting used to. Once you start using it, though, it's pretty nice for making code shorter and more parsable.
```suggestion
  def fetch_translation! = Gengo::SaveTranslatedText.run(translation_job_id)
```